src/yunohost/backup.py

```diff
@@ -1924,6 +1920,12 @@ def mount(self, restore_manager):
                          self._archive_file, exc_info=1)
             raise YunohostError('backup_archive_open_failed')
 
+        try:
+            files_in_archive = tar.getnames()
+            print(files_in_archive)
```

Comment:
some debug?

```suggestion
```

Reply:
Ugh yes sorry this was not optimal to do this at 3 am @.@